tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -346,6 +390,29 @@ public static void main(String[] args) {
     }
   }
 
+  private static Metadata createMetadataObj(long started, long completed, boolean status) {
```

Comment:
move static methods to Utils class.

Reply:
Yeah that's a good point. Will do.

Reply:
Fixed: https://github.com/linkedin/TonY/pull/79/commits/6ba7908b4d3312f24a810d97f8a82fa46e6c6a8e.

Reply:
Actually, on a second thought, see https://github.com/linkedin/TonY/pull/79#discussion_r233588794.